cmd/kubeadm/app/cmd/config.go

```diff
@@ -214,9 +215,59 @@ func NewCmdConfigImages(out io.Writer) *cobra.Command {
 		RunE:  cmdutil.SubCmdRunE("images"),
 	}
 	cmd.AddCommand(NewCmdConfigImagesList(out))
+	cmd.AddCommand(NewCmdConfigImagesPull())
 	return cmd
 }
 
+// NewCmdConfigImagesPull returns the `config images pull` command
+func NewCmdConfigImagesPull() *cobra.Command {
+	cfg := kubeadmapiv1alpha1.MasterConfiguration{}
+	kubeadmapiv1alpha1.SetDefaults_MasterConfiguration(&cfg)
```

Comment:
`kubeadmscheme.Scheme.Default(cfg)`
where `kubeadmscheme "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/scheme"`

Reply:
sure, seems reasonable. I need to understand this stuff better